atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2197,7 +2183,7 @@ private void throwIfValueChangedConflict(
             }
             if (!conflictingValues.containsKey(cell)) {
                 // This error case could happen if our locks expired.
-                preCommitRequirementValidator.throwIfPreCommitRequirementsNotMet(commitLocksToken, getStartTimestamp());
+                preCommitRequirementValidator.throwIfPreCommitRequirementsNotMet(getStartTimestamp());
```

Comment:
This is the actual place that validates the locks, and it calls back into the lock manager, so it's equivalent to passing it explicitly.

Reply:
Agreed.